Source/common/SNTConfigurator.m

```diff
@@ -181,6 +183,7 @@ - (instancetype)init {
       kMetricFormat : string,
       kMetricURL : string,
       kMetricExportInterval : number,
+      kMetricExtraLabels : string,
```

Comment:
You could declare this as a dictionary and avoid any parsing. You'll need to add an option for that on line 125 but then the getter on line 719 can just return the value directly.